includes/class-wc-payments-payment-method-messaging-element.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Class WC_Payments_Payment_Method_Messaging_Element
+ *
+ * @package WooCommerce\Payments
+ */
+
+declare( strict_types=1 );
```

Comment:
Not sure if the minimally supported PHP version for the client supports this.

Reply:
Addressed here [55c6ff2](https://github.com/Automattic/woocommerce-payments/pull/6522/commits/55c6ff25ddbeb78cc19c452d3aacf33eb1fbdefd) thanks!